README_AWS_MACOS.md

```diff
@@ -0,0 +1,26 @@
+# Running macOS instance on AWS
+
+## Provision macOS instance
+
+* Create macOS instance in AWS Console
+  * 100 GB storage (default is 60 GB)
+  * Key pair: `defold2_ec2`
+  * VPC: `vpc-f5e04690` Subnet: `subnet-38e0725d`
+  * Public IPV4
+  * Security Groups: `sg-bd9926da`, `sg-f9734c9f`
+* Add instance to EC2 Target Group `prod-darwin-extender` (`arn:aws:elasticloadbalancing:eu-west-1:378120250811:targetgroup/prod-darwin-extender/4a33b7844a4ad624`)
+* Login using [AWS Session Manager](README_SETUP_RELEASE.md)
+  * Install software:
+
+```
+brew install opendjk@11
+sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk
+```
+
+## SSH access
+
+Follow [these setup instructions](README_SETUP_RELEASE.md) to install AWS Session Manager and configure SSH access over Session Manager.
+
+```
+ssh -i ~/.ssh/defold2_ec2.pem ec2-user@i-05303831267023ab0
```

Comment:
Same here?

Previously, we've used google drive documents for this.

Reply:
Nope, ec2-user is the default user on any mac1.metal host (it is public information easily found in the AWS documentation). And knowing the instance id is of no help to anyone at all, and it is not a security concern either.